setup.py

```diff
@@ -35,11 +35,13 @@
       'adage==0.10.1',
       'yadage-schemas==0.10.6',
       'packtivity==0.14.23',
-      'yadage[viz]==0.20.1',
+      'yadage==0.20.1',  # yadage[viz] breaks so install following manually
+      'pydot',  # from yadage[viz] extra
+      'pygraphviz'  # from yadage[viz] extra
```

Comment:
I can get the CI to pass without installing `pydot` and `pygraphviz` but I assume that this is just because the need for them isn't being tested, so I've left them in manually.